runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogHeadImage/BlogHeadImage.module.scss

```diff
@@ -33,8 +39,7 @@
         position: absolute;
 
         bottom: -40px;
-        right: 70px;
-
+        right:  calc((100vw - $BACKGROUND-MAX-WIDTH)/2 + 40px);

```

Comment:
bruh whats that? xd

Reply:
I think this is the way to make logo move dynamically without adding additional element in leayout.